lib/nexmo_developer/app/presenters/topnav_item.rb

```diff
@@ -13,7 +13,7 @@ def title
 
   def css_classes
     classes = ['Vlt-tabs__link']
-    classes << 'Vlt-tabs__link_active' if @navigation == @name.downcase
+    classes << 'Vlt-tabs__link_active' if @navigation == @name.downcase || @navigation == @url.delete('/')
```

Comment:
I'm not sure about this, I would change `@name` in the config file to match the landing page file name (and the keys in the locale files too). that way we keep everything consistent.
WDYT?

Reply:
Done. PR opened in NDP to update the config files as well: https://github.com/Nexmo/nexmo-developer/pull/3043